definitions/checks/pulpcore/no_running_tasks.rb

```diff
@@ -0,0 +1,33 @@
+module Checks::Pulpcore
+  class NoRunningTasks < ForemanMaintain::Check
+    metadata do
+      for_feature :pulpcore
+      description 'Check for running pulpcore tasks'
+      tags :pre_upgrade
+      param :wait_for_tasks,
+        'Wait for tasks to finish or fail directly',
+        :required => false
+    end
+
+    def run
+      tasks = feature(:pulpcore).running_tasks
+      assert(tasks.empty?, failure_message(tasks.length),
+        :next_steps => calculate_next_steps)
```

Comment:
```suggestion
  assert(
    tasks.empty?,
    failure_message(tasks.length),
    :next_steps => calculate_next_steps
  )
```

I find this easier to read